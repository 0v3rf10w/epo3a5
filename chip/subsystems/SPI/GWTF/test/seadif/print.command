echo "Layout to Postscript printer command sequence" ; echo running getepslay -t Tmpspi26044 ; getepslay -t Tmpspi26044 ; echo printing Tmpspi26044.eps ; lpr Tmpspi26044.eps ; lpq ; rm -f Tmpspi26044.eps ; echo terminate >> seadif/print.out

