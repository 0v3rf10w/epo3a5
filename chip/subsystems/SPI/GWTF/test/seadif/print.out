Layout to Postscript printer command sequence
running getepslay -t Tmpspi26044
makegln: Temp file compression turned off
printing Tmpspi26044.eps
seadif/print.command: line 1: lpr: command not found
seadif/print.command: line 1: lpq: command not found
terminate
terminate
